### ISSUE TITLE
SMTP failed to send emails

### ISSUE THREAD

**Issue by mvandenbrink** [pocketbase/pocketbase#347]
Hi,

I upgraded yesterday to version 0.5.0, but it seems that the SMTP (gmail) doesn't work anymore. New users don't get an email  and after I started the server with the '--debug' option, there is no indication was goes wrong. When sending a test email, there is an error:

```
Failed to send the test email. Raw error: 
555 5.5.2 Syntax error. f4-20020a170906084400b0073c82539183sm2068370ejd.91 - gsmtp.
```

Version 0.3.3 works with the SMTP config that I used in 0.5.0.

**Comment by ganigeorgiev** [comment]
Hm, this is most likely the same as in https://github.com/pocketbase/pocketbase/discussions/346.

This is maybe a regression because there are indeed changes to how the `to` address is constructed as a workaround for the  spamassasin "TO_NO_BRKTS_HTML_ONLY" rule.

**Comment by mvandenbrink** [comment]
I was digging in the GO code and found out that indeed the spamassasin bluemonday code was removed. That could be the issue

**Comment by ganigeorgiev** [comment]
No, previously the "text" version of the email wasn't generated properly and resulted in increasing the spam score. The issue here is that some clients seems to reject emails when wrapped in angle brackets if there is missing name part. A pushed a revert commit in master and shortly will make a v0.5.1 release.

A revisited and more thoroughly tested text version of the email, which should also prevent the `TO_NO_BRKTS_HTML_ONLY` spam rule, will be added in v0.6.0 release.

**Comment by mvandenbrink** [comment]
Ah okay, thank you. I'll wait for the next release :-)

**Comment by mvandenbrink** [comment]
I can confirm that version 0.5.1 is working, thanks for the fast response.
